Assorted improvements for ERT

Shadowtail117 opened the pull request:
:cl:
admin: Admins can now specify a reason when dispatching an Emergency Response Team that will be shown to the team when they spawn in.
rscadd: The Emergency Response Team channel has its own key now: "O"/"o".
tweak: The Emergency Response Team welcome text has been modified to be more clear on who the leader is and your role in the game. It also includes the reason specified by the admin that dispatched it.
maptweak: Added an all-in-one telecomms machine to the ERT shuttle that allows them to speak on their own private frequency at the base, in transit, and on the Torch (or anywhere their shuttle is). It is not advised to mess with it even if you have the tools to do so.
tweak: Emergency Response Team headsets now have their default channel set to their private frequency. This can be manually changed back to the common frequency, but doing so is irreversible!
maptweak: Added a reagent grinder to the medical section of the Emergency Response Team base.
/:cl:

Changing the headset's default frequency is the change I'm most concerned about since it blocks off access to the Common channel unless they change it, and even then that change is irreversible due to how headsets work. Some feedback on how best to approach this would be appreciated.

Mockup join text:
![Annotation 2020-07-29 124620](https://user-images.githubusercontent.com/25853190/88845826-93e9dc80-d199-11ea-8f9c-796d4073e670.png)

PsiOmegaDelta commented on code/game/antagonist/outsider/ert.dm:
```diff
@@ -26,19 +26,25 @@ GLOBAL_DATUM_INIT(ert, /datum/antagonist/ert, new)
 
+	if(reason == "")
+		return
+	to_chat(player.current, SPAN_BOLD(FONT_LARGE("You have been summoned to \the [GLOB.using_map.station_name] for the following reason: " + SPAN_NOTICE(reason))))
```
Why not
```
if(reason)
    to_chat(player.current, SPAN_BOLD(FONT_LARGE("You have been summoned to \the [GLOB.using_map.station_name] for the following reason: " + SPAN_NOTICE(reason))))
```
?
There isn't a lot of indentation going on which you'd avoid, and generally checking for non-negated cases is preferable from a code-reading standpoint.

Shadowtail117 replied:
I wasn’t entirely sure how Byond does null checking on strings; I’ll make this change soon.

SierraKomodo commented on code/game/response_team.dm:
```diff
@@ -28,13 +28,24 @@ var/can_call_ert
+		if(alert("You did not input a reason. Continue anyway?",,"Yes", "No") != "Yes")
+			return
+		if(send_emergency_team)
+			to_chat(usr, "<span class='danger'>Looks like somebody beat you to it!</span>")
```
Use `SPAN_DANGER("Message here")` instead of `"<span class='danger'>Message Here</span>"`

SierraKomodo commented on code/game/response_team.dm:
```diff
@@ -28,13 +28,24 @@ var/can_call_ert
+		message_admins("[key_name_admin(usr)] is dispatching an Emergency Response Team.", 1)
+	else
+		if(send_emergency_team)
+			to_chat(usr, "<span class='danger'>Looks like somebody beat you to it!</span>")
```
Use `SPAN_DANGER("Message here")` instead of `"<span class='danger'>Message Here</span>"`

Shadowtail117 commented on code/game/antagonist/outsider/ert.dm:
```diff
@@ -4,12 +4,10 @@ GLOBAL_DATUM_INIT(ert, /datum/antagonist/ert, new)
-		and before taking extreme actions, please try to also contact the administration! \
-		Think through your actions and make the roleplay immersive! <b>Please remember all \
-		rules aside from those without explicit exceptions apply to the ERT.</b>"
+	antag_text = "You are an <b>anti</b>-antagonist! Within the rules, try to save the ship and its crew from the ongoing crisis. \
+				 Try to make sure the other players have <i>fun</i>, and if you are confused or at a loss, always adminhelp. \
+				 You should also contact the staff before taking any extreme actions. \
+				 <b>Remember that all rules outside of those with explicit exceptions apply to the ERT!</b>"
```
Using inline tags like this isn't ideal but converting it to span macros would muddle up the text for very little gain.

SierraKomodo commented on code/game/response_team.dm:
```diff
@@ -28,13 +28,24 @@ var/can_call_ert
+		if(send_emergency_team)
+			to_chat(usr, "<span class='danger'>Looks like somebody beat you to it!</span>")
+			return
+		message_admins("[key_name_admin(usr)] is dispatching an Emergency Response Team for the reason: [reason].", 1)
```
This entire section can probably be simplified down to something like the following:

```dm
	var/reason = input("What is the reason for dispatching this Emergency Response Team?", "Dispatching Emergency Response Team")

	if(!reason && alert("You did not input a reason. Continue anyway?",,"Yes", "No") != "Yes")
		return

	if(send_emergency_team)
		to_chat(usr, SPAN_DANGER("Looks like somebody beat you to it!"))
		return

	message_admins("[key_name_admin(usr)] is dispatching an Emergency Response Team for the reason: [reason].", 1)
```

Shadowtail117 replied:
"XXX is dispatching an Emergency Response Team for the reason:" looks bad which is why I added a separate log for if there is no reason. Though yes, it is unoptimized code at its core. I'll make these changes soon.

SierraKomodo reviewed (changes requested):
Couldn't test the radio when off-base because the shuttle launch button is greyed out for some reason. Otherwise, see individual comments.

Shadowtail117 committed:
Assorted improvements for ERT

- Admins can specify a reason when calling
- ERT channel has its own hotkey
- Welcome text is clearer
- Add all-in-one telecomms unit on the ERT shuttle
- ERT headset default frequency changed
- Add reagent grinder to ERT medical base
